### ISSUE TITLE
Remove the Powershell Hack 

### ISSUE THREAD

**Issue by cleemullins** [microsoft/botbuilder-dotnet#1872]
Once the Powershell version is published and on Azure Devops, remove the hack from our build script:

# Download temporary version of Archive module that fixes issue on macOS/Linux with path separator
Invoke-WebRequest -Uri "https://raw.githubusercontent.com/PowerShell/Microsoft.PowerShell.Archive/master/Microsoft.PowerShell.Archive/Microsoft.PowerShell.Archive.psm1" -OutFile .\archive.psm1
Import-Module .\archive.psm1

**Comment by cleemullins** [comment]
The code is found here:
https://github.com/PowerShell/Microsoft.PowerShell.Archive/tree/master/Microsoft.PowerShell.Archive

**Comment by anmenaga** [comment]
Fixed module [was released on PS Gallery](https://www.powershellgallery.com/packages/Microsoft.PowerShell.Archive/1.2.3.0) with version `1.2.3.0`.
To use it:
`Install-Module Microsoft.PowerShell.Archive -MinimumVersion 1.2.3.0 -Force -Repository PSGallery
Import-Module Microsoft.PowerShell.Archive -RequiredVersion 1.2.3.0 -Force`

**Comment by cleemullins** [comment]
@BruceHaley  - Bruce, can you remove our "Very ugly" powershell hack? It's possible the latest is already on the Azure Devops server(s), and if not we can do the install/import shown here rather than the hack we have now...

**Comment by cleemullins** [comment]
To tell what version of powershell is running, run this in a Powershell step on one of our Azure DEvops VMs and see what comes back:

$psversiontable

Get-Module Microsoft.PowerShell.Archive -ListAvailable

# this should list archive module with version 1.2.3.0

```
   
      PS C:\> $psversiontable
Name                           Value
----                           -----
PSVersion                      6.2.0-preview.3
PSEdition                      Core
GitCommitId                    6.2.0-preview.3
OS                             Microsoft Windows 10.0.14393
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
PS C:\> Get-Module Microsoft.PowerShell.Archive -ListAvailable
    Directory: C:\Users\anmenaga\Documents\PowerShell\Modules
ModuleType Version    Name                                PSEdition ExportedCommands
---------- -------    ----                                --------- ----------------
Manifest   1.2.3.0    Microsoft.PowerShell.Archive        Desk      {Compress-Archive, Expand-Archive}
    

    
  

  







```

**Comment by BruceHaley** [comment]
No update yet in Azure DevOps task. The current version is 1.0.1.0:

Name                           Value |  
-- | --
  | ----                           ----- |  
  | PSVersion                      5.1.17763.503 |  
  | PSEdition                      Desktop |  
  | PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...} |  
  | BuildVersion                   10.0.17763.503 |  
  | CLRVersion                     4.0.30319.42000 |  
  | WSManStackVersion              3.0 |  
  | PSRemotingProtocolVersion      2.3 |  
  | SerializationVersion           1.1.0.1 |  
  |   |  
  | Name              : Microsoft.PowerShell.Archive |  
  | Path              : C:\windows\system32\WindowsPowerShell\v1.0\Modules\Microsoft.PowerShell.Archive\Microsoft.PowerShel |  
  | l.Archive.psd1 |  
  | Description       : |  
  | Guid              : eb74e8da-9ae2-482a-a648-e96550fb8733 |  
  | Version           : 1.0.1.0 |  
  | ModuleBase        : C:\windows\system32\WindowsPowerShell\v1.0\Modules\Microsoft.PowerShell.Archive |  
  | ModuleType        : Manifest |  
  | PrivateData       : |  
  | AccessMode        : ReadWrite |  
  | ExportedAliases   : {} |  
  | ExportedCmdlets   : {} |  
  | ExportedFunctions : {[Compress-Archive, Compress-Archive], [Expand-Archive, Expand-Archive]} |  
  | ExportedVariables : {} |  
  | NestedModules     : {}

**Comment by BruceHaley** [comment]
The first task in this daily build reports what version is currently available. We just look at the latest log output to check it: https://fuselabs.visualstudio.com/SDK_v4/_build?definitionId=277&_a=summary

**Comment by BruceHaley** [comment]
PR: The next path separator fix hack for Linux NuGet installs #2108 https://github.com/microsoft/botbuilder-dotnet/pull/2108

**Comment by sgellock** [comment]
issue is fixed and verified
